app/src/main/java/com/example/genericapplication/services/VODProviderService.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.genericapplication.services
+
+import com.example.genericapplication.factories.DotenvFactory
+
+class VODProviderService {
```

Comment:
@DiegoA-coder why we need a VOD provider service if the VideoProviderService includes the VOD case?